src/main/kotlin/blackjack/domain/Denomination.kt

```diff
@@ -0,0 +1,52 @@
+package blackjack.domain
+
+enum class Denomination(private val primaryScore: Score, private val secondaryScore: Score) {
+
+    ACE(primaryScore = Score(11), secondaryScore = Score(1)),
+    TWO(Score(2)),
+    THREE(Score(3)),
+    FOUR(Score(4)),
+    FIVE(Score(5)),
+    SIX(Score(6)),
+    SEVEN(Score(7)),
+    EIGHT(Score(8)),
+    NINE(Score(9)),
+    TEN(Score(10)),
+    JACK(Score(10)),
+    QUEEN(Score(10)),
+    KING(Score(10));
+
+    constructor(score: Score) : this(score, score)
+
+    fun calculateScore(otherDenominations: List<Denomination>): Score {
+        val denominations = listOf(this, *otherDenominations.toTypedArray())
```

Comment:
저는 아래가 더 깔끔해보이는데, 어떻게 생각하시는가요?
```suggestion
        val denominations = listOf(this) + otherDenominations
```

Reply:
`Collection` 확장함수로 `plus` 연산자를 만들어두었군요 🙇‍♂️